src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -23,8 +23,8 @@ public sealed class DurableOrchestrationContext : DurableOrchestrationContextBas
         private const string DefaultVersion = "";
         private const int MaxTimerDurationInDays = 6;
 
-        private readonly Dictionary<string, object> pendingExternalEvents =
-            new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+        private readonly Dictionary<string, ICollection> pendingExternalEvents =
+            new Dictionary<string, ICollection>(StringComparer.OrdinalIgnoreCase);

```

Comment:
Let's switch to `Dictionary<string, Queue>`.

Reply:
Thank you! I will change it.